src/adhocracy_mercator/adhocracy_mercator/workflows/test_mercator2.py

```diff
@@ -0,0 +1,283 @@
+from pyramid import testing
+from pytest import mark
+from pytest import fixture
+from webtest import TestResponse
+
+from adhocracy_core.utils.testing import add_resources
+from adhocracy_core.utils.testing import do_transition_to
+
+@fixture
+def integration(config):
+    config.include('adhocracy_core.events')
+    config.include('adhocracy_core.content')
+    config.include('adhocracy_mercator.workflows')
+
+
+def _post_proposal(app_user, path='/') -> TestResponse:
+    from adhocracy_mercator.resources.mercator2 import IMercatorProposal
+    resp = app_user.post_resource(path, IMercatorProposal, {})
+    return resp
+
+
+def _post_comment_item(app_user, path='') -> TestResponse:
+    from adhocracy_core.resources.comment import IComment
+    resp = app_user.post_resource(path, IComment, {})
+    return resp
+
+
+@mark.functional
+class TestMercator2:
+
+    @fixture
+    def process_url(self):
+        return '/organisation/advocate-europe2'
+
+    @fixture
+    def proposal0_url(self):
+        return '/organisation/advocate-europe2/proposal_0000000'
+
+
+    def test_create_resources(self,
+                              registry,
+                              datadir,
+                              process_url,
+                              app,
+                              app_admin):
+        json_file = str(datadir.join('resources.json'))
+        add_resources(app, json_file)
+        resp = app_admin.get(process_url)
+        assert resp.status_code == 200
+
+    def test_set_participate_state(self, registry, app, process_url, app_admin):
+        resp = app_admin.get(process_url)
+        assert resp.status_code == 200
+
+        resp = do_transition_to(app_admin,
+                                process_url,
+                                'announce')
+        assert resp.status_code == 200
+
+        resp = do_transition_to(app_admin,
+                                process_url,
+                                'participate')
+        assert resp.status_code == 200
+
+    def test_participate_participant_creates_proposal(self,
+                                                      registry,
+                                                      app,
+                                                      process_url,
+                                                      app_participant):
+        resp = _post_proposal(app_participant, path=process_url)
```

Comment:
Up to now we added all parameters for test methods in one line instead of seperated lines to 
save same space. To we want to change this?